mypy_django_plugin/lib/helpers.py

```diff
@@ -331,15 +339,15 @@ def check_types_compatible(
     api.check_subtype(actual_type, expected_type, ctx.context, error_message, "got", "expected")
 
 
-def add_new_sym_for_info(info: TypeInfo, *, name: str, sym_type: MypyType) -> None:
+def add_new_sym_for_info(info: TypeInfo, *, name: str, sym_type: MypyType, no_serialize: bool = False) -> None:
```

Comment:
Probably this is the cause for other errors like the one with `WithAnnotations` 🤔 

We should disallow serializing it as well.

Reply:
Yeah, that might be a reason